docs/sources/panels-visualizations/visualizations/time-series/index.md

```diff
@@ -178,6 +178,10 @@ The following example shows three series: Min, Max, and Value. The Min and Max s
 
 Options under the axis category change how the X and Y axes are rendered. Some options do not take effect until you click outside of the field option box you are editing. You can also or press `Enter`.
 
+### Time zone
+
+Set the desired time zone(s) to display along the X-axis.
```

Comment:
```suggestion
Set the desired time zone(s) to display along the x-axis.
```

Reply:
re: x / y axis capitalization, would you like me to go through the rest of the time series doc file to lower-case the other instances of `X-axis` + `Y-axis`? I think default in the file before was to capitalize them (which is convention I ended up following for added sections)